src/Plugins/Only.php

```diff
@@ -33,6 +33,8 @@ public function shutdown(): void
         if (file_exists($lockFile)) {
             unlink($lockFile);
         }
+
+        echo 'The tests executed were tests filtered using <href=https://pestphp.com/docs/filtering-tests#content-only>only</>';
```

Comment:
Can you inject the output, like other plugins are doing, and use the info stuff?